pkg/utils/units/units.go

```diff
@@ -0,0 +1,44 @@
+package units
+
+type ByteUnit uint64
+
+const (
+	Bytes ByteUnit = 1 << (10 * iota)
+	Kilobytes
+	Megabytes
+	Gigabytes
+	Terabytes
+)
+
+// String returns the string representation of the byte unit.
+func (b ByteUnit) String() string {
+	switch b {
+	case Kilobytes:
+		return "KB"
+	case Megabytes:
+		return "MB"
+	case Gigabytes:
+		return "GB"
+	case Terabytes:
+		return "TB"
+	default:
+		return "B"
+	}
```

Comment:
[Technically these aren't kilo-, mega-, giga-, tera- bytes](https://www.techtarget.com/searchstorage/definition/Kibi-mebi-gibi-tebi-pebi-and-all-that?Offer=abt_pubpro_AI-Insider). They are in fact kibi-, mebi-, gibi-, and tebi- bytes because of the byte shift values. I've found that the cloud providers are moving towards more precision with these units and it is useful for us to do so as well. At the very least, it ups our nerd factor ;-)

```suggestion
	switch b {
	case Kilobytes:
		return "KiB"
	case Megabytes:
		return "MiB"
	case Gigabytes:
		return "GiB"
	case Terabytes:
		return "TiB"
	default:
		return "B"
	}
```